### ISSUE TITLE
Minor issues with Klonoa: Moonlight Museum (English translation)

### ISSUE THREAD

**Issue by clairofmemories** [xdanieldzd/StoicGoose#14]
I've been using StoicGoose to play through the fan translation of Klonoa: Moonlight Museum, and for the most part it runs great, but there are 2 issues I've noticed, neither of which are major but they're still just kind of inconvenient (This is all on the non-gl version)

The first one is that sometimes the music seems to glitch out, it stutters for a while and then stops, and the only way to fix it that I found was to close and then reopen the emulator. This one isn't too big of an issue since you don't need the music, the levels aren't too long, and you get a chance to save after every one, it's just kind of annoying

The other issue is that the game freezes whenever you die, and like the music bug, it's really just more of a minor inconvenience since whenever you restart the game it lets you continue from the start of the last room you entered

I don't know if these also happen a non patched version of the game, but if you would like for me to test that out I can. The game is playable despite these, and it's been a very good emulator so far, i appreciate your work!

**Comment by clairofmemories** [comment]
Update:
I have now finished the main part of the game, all except for the EX levels. I realized the game was running in WonderSwan Color mode instead of WonderSwan mode, and fixing that seemed to fix the audio issues. I did not find any other issues with it, but the freezing after losing a life makes specifically some of the EX levels very annoying due to their difficulty.

**Comment by xdanieldzd** [comment]
Yeah, can confirm that the game freezes when losing a life even with the original, unpatched ROM. I haven't noticed the music glitching out, but I just might've not waited long enough for that to happen.

I'll try to look into the freezing bug at least, once I get the chance.
